.github/workflows/cf-deploy.yml

```diff
@@ -23,8 +23,8 @@ jobs:
             echo "::set-output name=env_name::green-dev"
             echo "::set-output name=env_file::src/environments/environment.dev.ts"
             echo "::set-output name=build_script::npm run build"
-          elif [ "${{ github.ref }}" = "refs/heads/release" ]; then
-            echo "::set-output name=env_name::prod"
+          elif [ "${{ github.ref }}" = "refs/heads/green-release" ]; then
+            echo "::set-output name=env_name::green-release"
```

Comment:
@dawidgil is this the right `env_name`?

Reply:
Actually, I think this is okay. I think this just configures the GitHub environment name, as far as I can tell

Reply:
Yeah, this is Github env stuff.